src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -665,13 +665,15 @@ public boolean isFilterChangelog() {
       return filterChangelog;
     }
 
+    @Override
+    public void buildEnvVars(AbstractBuild<?,?> build, Map<String,String> env) {
+        buildEnvironment(build, env);
+    }
+
     /**
      * Sets the <tt>SVN_REVISION_n</tt> and <tt>SVN_URL_n</tt> environment variables during the build.
      */
-    @Override
-    public void buildEnvVars(AbstractBuild<?, ?> build, Map<String, String> env) {
-        super.buildEnvVars(build, env);
-
+    public void buildEnvironment(Run<?, ?> build, Map<String, String> env) {
```

Comment:
I would add info about target Jenkins core version to Javadoc. maybe with "TODO: override when the baseline is above N"